neps/nep-0141.md

```diff
@@ -293,35 +295,38 @@ Finalize an `ft_transfer_call` chain of cross-contract calls.
 
 The `ft_transfer_call` process:
 
- 1. Sender calls `ft_transfer_call` on FT contract
- 2. FT contract transfers `amount` tokens from sender to receiver
- 3. FT contract calls `ft_on_transfer` on receiver contract
- 4+. [receiver contract may make other cross-contract calls]
- N. FT contract resolves promise chain with `ft_resolve_transfer`, and may
+1.  Sender calls `ft_transfer_call` on FT contract
+2.  FT contract transfers `amount` tokens from sender to receiver
+3.  FT contract calls `ft_on_transfer` on receiver contract
+    4+. [receiver contract may make other cross-contract calls]
```

Comment:
The formatting here and the next line will be broken. I guess we should just change it to 4. And 5.

Reply:
Most of things were auto-formatted. Yes, will need to go through the diff.